lib/src/require.dart

```diff
@@ -0,0 +1,16 @@
+import 'bool_extensions.dart';
+
+/// Util for checking
+/// `since 0.7.0`
+class Require {
+  Require._();
+
+  /// Throws an exception with [message] if [check] is [false]
+  /// `since 0.7.0`
+  static bool require(bool check, String message) {
+    if (check.isFalse) {
+      throw Exception(message);
```

Comment:
```
/// Creating instances of [Exception] directly with `Exception("message")`
/// is discouraged in library code since it doesn't give users a precise
/// type they can catch. It may be reasonable to use instances of this
/// class in tests or during development.
```